iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/TreeFactory.java

```diff
@@ -167,14 +167,25 @@ public TargetScopeDeclaration targetScopeDeclaration(SyntaxToken keyword, Syntax
     return new TargetScopeDeclarationImpl(keyword, equals, expression);
   }
 
-  public ParameterDeclaration parameterDeclaration(SyntaxToken keyword, Identifier name, StringLiteral typeExpression, Optional<SyntaxToken> equ,
+  public ParameterDeclaration parameterDeclaration(
+    Optional<List<Decorator>> decorators,
+    SyntaxToken keyword,
+    Identifier name,
+    StringLiteral typeExpression,
+    Optional<SyntaxToken> equ,
     Optional<Expression> defaultValue) {
-    return new ParameterDeclarationImpl(keyword, name, typeExpression, equ.orNull(), defaultValue.orNull());
+    return new ParameterDeclarationImpl(decorators.orNull(), keyword, name, typeExpression, equ.orNull(), defaultValue.orNull());
```

Comment:
According to [recent dicsussion on slack](https://sonarsource.slack.com/archives/C05E4T1KGH3/p1689346311891899), we prefer to store Col`lections.emptyList()` instead of `null` for collections.